Shared/Samples/Create and edit geometries/CreateAndEditGeometriesView.swift

```diff
@@ -364,6 +390,7 @@ private class GeometryEditorModel: ObservableObject {
     func stop() {
         geometryEditor.stop()
```

Comment:
After reading the implementation details, I like the idea to showcase that `stop` method returns the final geometry, because it might be something that a user won't discover by merely following this sample, without reading the doc.

Instead of hiding then removing the graphic being edited, can you modify the geometry instead? Thanks.